build/_build.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
There is no need to specify multiple targets for the build project.
We can use just the latest version here.

Reply:
Changed it to only target .net 8